packages/editor-ui/src/stores/segment.store.ts

```diff
@@ -27,13 +29,28 @@ export const useSegment = defineStore('segment', () => {
 	const nodeTypesStore = useNodeTypesStore();
 	const workflowsStore = useWorkflowsStore();
 	const settingsStore = useSettingsStore();
+	const usersStore = useUsersStore();
 
 	const track = (eventName: string, properties?: ITelemetryTrackProperties) => {
 		if (settingsStore.telemetry.enabled) {
 			window.analytics?.track(eventName, properties);
 		}
 	};
 
+	const page = (category: string, name: string, properties?: ITelemetryTrackProperties) => {
+		if (settingsStore.telemetry.enabled) {
+			window.analytics?.page(category, name, properties);
+		}
+	};
+
+	const identify = () => {
+		const userId = usersStore.currentUserId;
+
+		if (settingsStore.telemetry.enabled && userId) {
+			window.analytics?.identify(userId);
```

Comment:
is this the same id we are using in the hooks? I imagine in cloud we are using the cloud id